algorithms/x25519.js

```diff
@@ -61,8 +76,9 @@ export async function kekFromStaticPeer({ephemeralKeyPair, staticPublicKey}) {
     throw new Error(
       `"staticPublicKey.type" must be "${KEY_TYPE}".`);
   }
-  const publicKeyBase58 = staticPublicKey.publicKeyMultibase.slice(1);
-  const remotePublicKey = base58.decode(publicKeyBase58);
+  const remotePublicKey = multibaseDecode(
+    MULTICODEC_X25519_PUB_HEADER, staticPublicKey.publicKeyMultibase
+  );
```

Comment:
```suggestion
  const remotePublicKey = multibaseDecode(
    MULTICODEC_X25519_PUB_HEADER, staticPublicKey.publicKeyMultibase);
```

Reply:
@dlongley - I thought our style guide preferred the former?

Reply:
Nope, and we've got reams and reams of code where the only thing you drop are the params. You only drop the parenthesis if you've got a multiline object that needs to be closed, eg:

```js
foo({
  bar: 'baz'
});
```

Reply:
Updated here d254eb8